### PR TITLE
Now the gym leaders name shows up on the button

### PR DESCRIPTION
**Pull request by Jaaslet**
Example: Pewter City's gym's button now have the text "Brock's gym"

### CODE REVIEW

File: src/scripts/gym/Gym.ts [hunk 1]
```diff
@@ -73,7 +73,11 @@ class Gym extends TownContent {
         super(requirements);
         this.flags.quest = quest;
         this.flags.achievement = achievement;
-        this.buttonText = leaderName.replace(/\d/g,'');
```
Comment (RedSparr0w) [inline src/scripts/gym/Gym.ts:76]: We should probably continue to replace the `\d` as we have `Bruno2` (and maybe others, not too sure)

Comment (Jaaslet) [reply src/scripts/gym/Gym.ts:76]: Yeah, i am stupid. Did not think it was used for anything.
It's back now

File: src/scripts/gym/Gym.ts [hunk 2]
```diff
@@ -73,7 +73,11 @@ class Gym extends TownContent {
...
+        if (!town.includes('Elite') && !town.includes('Champion')) {
+            this.buttonText = `${leaderName}'s gym`;
```
Comment (RedSparr0w) [inline src/scripts/gym/Gym.ts:77]:
```suggestion
            this.buttonText = `${leaderName.replace(/\d/g,'')}'s gym`;
```

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

**Pull request merged**